recursion/print_increasing_decreasing.js

```diff
@@ -0,0 +1,10 @@
+const printDecreasing = (n, i) => {
+    if( i > n ){
+        return;
+    }
+    console.log(i);
```

Comment:
it will print in increasing order remove it

Reply:
Yes, its intended. Modified ques as branch name says